aiven-python/__main__.py

```diff
@@ -0,0 +1,25 @@
+import pulumi
+import pulumi_aiven as aiven
+
+kafka1 = aiven.Kafka("kafka1",
+    project="<YOUR_AIVEN_PROJECT_NAME>",
+    cloud_name="google-europe-west1",
+    plan="business-4",
+    service_name="kafka-gcp-eu",
```

Comment:
Similar to https://github.com/pulumi/templates/pull/480#discussion_r1067569171, I'd suggest we make these configurable.